ports/raspberrypi/common-hal/rp2pio/StateMachine.c

```diff
@@ -361,12 +367,6 @@ void common_hal_rp2pio_statemachine_construct(rp2pio_statemachine_obj_t *self,
                 out_loaded = true;
             }
         }
```

Comment:
```suggestion
        }
        if (instruction == pio_instr_bits_jmp) {
            uint16_t condition = (full_instruction & 0x00e0) >> 5;
            if (condition == 0x6 && jmp_pin == NULL) {
                mp_raise_ValueError_varg(translate("Missing jmp_pin. Instruction %d jumps on pin"), i);
            }
        }
```
An error should be raised if no `jmp_pin` is provided in this case.

Reply:
Good suggestion @microDev1 - thank you :-)
I have made the change, tested it and pushed up a new commit.